test/e2e/lib/framework/pageSetups.ts

```diff
@@ -37,11 +37,19 @@ export function asyncSetup(options: SetupOptions) {
         window.addEventListener('load', () => {
           const logs = document.createElement('script')
           logs.src = './datadog-logs.js'
-          logs.onload = () => {
-            DD_LOGS.init(${formatLogsOptions(options.logs)})
-          }
           document.getElementsByTagName('head')[0].appendChild(logs)
         })
+
+        window.DD_LOGS = window.DD_LOGS || {
+          onReady: function(c) {
+            DD_LOGS.q.push(c)
+          },
+          q: [],
+        }
+
+        DD_LOGS.onReady(function() {
+          DD_LOGS.init(${formatLogsOptions(options.logs)})
+        })
```

Comment:
it could be nice to use a setup as close as possible to what we will provide to customers.
what do you think?

Reply:
I agree, but I thought this was what will be provided to customers? I took it from your document https://docs.google.com/document/d/1EsiF_TmXs0exzBWBeowUaupoAm8Ra5MlH6dLyHLkfNM/edit#heading=h.a5fyuuv435sa

Reply:
oh ok, in this section of the doc only the buffer mechanism is described not the loading of the SDK.
It could be nice to have a compact snippet like in the other sections of the doc.

Reply:
I changed the snippet, let me know what you think!